cfgov/preprocessed/css/organisms/full-width-well.less

```diff
@@ -0,0 +1,45 @@
+/* topdoc
+  name: Bullet Points for full width text
+  family: cfgov-organisms
+  patterns:
+    - name: Default example
+      markup: |
+        <div class="o-full-width-text">
+            <ul>
+              <li>Gray Well</li>
+              <li>Well's content</li>
+            </ul>
+        </div>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          .o-full-width-text
+            ul
+              li
+  tags:
+    - cfgov-organisms
+  notes:
+    - Since content will be entered by content managers,
+      these styles are applied on the ul li elements directly when needed.
+*/
+
+.o-full-width-text {
+    ul {
+        padding-left: 30px;
+
+        li {
+          @font-size: 16px;
+          @offset: unit(15px / @font-size, em);
+          @size: unit(22px / @base-font-size-px, em);
+          .custom-bullet( @text:"\25AA", @color: @black, @offset: @offset, @size: @size );
+          // Spacing has been eyeballed to be 30px.
+          margin-bottom: unit( 21px / @font-size, em );
```

Comment:
We do have a lot that are the direct values, which is too bad I think since having it with the variable minus an offset means it would be easy to find where the default padding is used in a find operation, and should the padding ever change it could be changed directly in the variable. We do have it with the variable in [some cases]( https://github.com/cfpb/cfgov-refresh/blob/flapjack/cfgov/preprocessed/css/nav-secondary.less#L205), but yeah it's a mix of both in the codebase. Well for this PR I'd say either is fine. If you want to leave it with the eyeballed comment that works since there are plenty of other spots using that.